react/details/README.md

```diff
@@ -0,0 +1,93 @@
+# Details Widget
+
+## Widget API
+
+### Attributes
+
+- **locale** (default: `en`)
+- **entityName**
+- **entityElementId**
+
+
+## i18n
+
+To add a new locale:
+
+- add a new language file under `src/i18n/locales/[newLocaleName].json`
+- edit `src/i18n/locales/index.js` to add new language file
+
+## API
+
+API helper functions have been added at `src/api/entity-api.js`. 4 functions are provided - `get()`, `post()`, `put()`, `delete()`.
```

Comment:
The widget does not need all the API functions. Do we want to provide them regardless?

Reply:
If these widgets are a starting point for 3rd party developers - this could help a lot to push forward fast with POCs. But it's up to what is the end goal with these